release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -654,13 +654,23 @@ For example, the plugin can highlight unexpected differences, such as mismatched
 You can use the `cluster-compare` plugin in development, production, and support scenarios.
 
 For more information about the `cluster-compare` plugin, see xref:../scalability_and_performance/cluster-compare/understanding-the-cluster-compare-plugin.adoc#cluster-compare-overview_understanding-cluster-compare[Overview of the cluster-compare plugin].
+
 [id="ocp-release-notes-node-tuning-operator-deferred-updates_{context}"]
 ==== Node Tuning Operator: Deferred Tuning Updates
 
 In this release, the Node Tuning Operator introduces support for deferring tuning updates. Administrators can schedule updates to be applied during a maintenance window with this feature.
 
 For more information, see xref:../scalability_and_performance/using-node-tuning-operator.adoc#defer-application-of-tuning-changes_node-tuning-operator[Deferring application of tuning changes].
 
+[id="ocp-release-notes-scalability-and-performance-nto-platforms_{context}"]
+==== Node Tuning Operator platform detection
+
+With this release, when you apply a performance profile, the Node Tuning Operator detects the platform and configures kernel arguments and other platform-specific options accordingly. This release adds support for detecting the following platforms:
+* AMD x86
+* ARM aarch64
+* Intel x86
+
```

Comment:
```suggestion
With this release, when you apply a performance profile, the Node Tuning Operator detects the platform and configures kernel arguments and other platform-specific options accordingly. This release adds support for detecting the following platforms:

* AMD x86
* ARM aarch64
* Intel x86

```

After line no 668, one new line is needed so that the bullet list renders properly in the preview. 

See preview: https://88630--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-18-release-notes.html#ocp-release-notes-scalability-and-performance-nto-platforms_release-notes

Reply:
Oops, nice catch. Thanks!